### ISSUE TITLE
FoundryVTT Error with GCS 5 Imported Files

### ISSUE THREAD

**Issue by GambitDeclined** [crnormand/gurps#1755]
Reproducible error with FoundryVTT 10.291 and GURPS 4e Game Aid (Unofficial) 0.15.27 import from GCS 5. Some weapons return a “No melee or ranged named error such as “No melee or ranged attached named ‘Laser Pistol’ found on Jeff Green” when attempting to assign damage to target.

This error is not seen in GCS 4 files.
 
[Reproducible error with FoundryVTT 10.pdf](https://github.com/crnormand/gurps/files/10614177/Reproducible.error.with.FoundryVTT.10.pdf)

**Comment by crnormand** [comment]
Can you attach a copy of the gca5 file so I can import/export it here and attempt to reproduce?

**Comment by GambitDeclined** [comment]
Here you go.
[02052023a Jeff Green.zip](https://github.com/crnormand/gurps/files/10632303/02052023a.Jeff.Green.zip)

**Comment by crnormand** [comment]
Well crap.   I finally got back to this..... and I can't reproduce the error with the attached character sheet.  But I am still looking into it.

**Comment by crnormand** [comment]
![image](https://user-images.githubusercontent.com/8931036/224564037-5a23a6fa-2228-4528-b8bb-fe53e66c7c39.png)

The attached character did not have a Laser Pistol, so I got one from the Basic Set library, added it and saved.   I imported into Foundry, and then rolled 2 attacks from the character sheet, and then 2 more from the Token Action HUD module.   I also rolled damage from the character sheet and the TAH module.    

If you are still seeing an issue... you might need to create a new actor and import "Jeff".    Transfer over any Foundry items you might be using, and then delete the old actor.

**Comment by GambitDeclined** [comment]
I recreated my Jeff character. And the laser pistol is working correctly, but not the grav needler from UT. I also created a test character with just the UT laser pistol and the grav needler with the same result. 

I'm beginning to suspect corruption in FoundryVTT or perhaps the World. 

![gravNeedler](https://user-images.githubusercontent.com/46287567/224861389-e2b0d34d-7133-460f-a915-57ab4f450bd0.jpg)
